lib/src/message_handlers.dart

```diff
@@ -160,3 +162,19 @@ class PoiDeselectedMessageHandler extends PoiSelectionMessageHandler {
         ?.call(OnPoiDeselectedResult(poi: poi));
   }
 }
+
+class SpeakAloudTextMessageHandler implements MessageHandler {
+  @override
+  void handleMessage(
+      MapViewController mapViewController, Map<String, dynamic> payload) async {
+    if (mapViewController._onSpeakAloudTextCallback == null &&
+        mapViewController._onSpeakAloudTextCallback == null) {
+      return;
```

Comment:
Temos un "AND" coa misma comprobación dúas veces?

Reply:
De todas formas, ao invocar a callback co operador de nulabilidade mapViewController._onSpeakAloudTextCallback**?** non faría falta ese if.

Reply:
correxido